demos/view-job-filters/README.md

```diff
@@ -31,7 +31,6 @@ jenkins:
             neverBuilt: true
         - jobTypeFilter:
```

Comment:
is it working without the type though? better to just remove this filter if it's not compatible with jcasc

Reply:
Oh sorry, missed that :sweat_smile:  thanks for pointing it out